infrastructure/helm-chart/charts/controller/templates/service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: controller-info
```

Comment:
I think that `airy-controller` would be a more appropriate name for the service.